arrocke/gloss-translation
# feat: favicon and tab titles

## What has changed

- Each page in the app displays [this favicon](https://private-user-images.githubusercontent.com/11968925/302839902-956e2b0d-1bfb-40f2-b3b9-bf5e8275a858.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDcyNzk2NjIsIm5iZiI6MTcwNzI3OTM2MiwicGF0aCI6Ii8xMTk2ODkyNS8zMDI4Mzk5MDItOTU2ZTJiMGQtMWJmYi00MGYyLWIzYjktYmY1ZTgyNzVhODU4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMDclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjA3VDA0MTYwMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM0NDgyZWVmNDExNTJmZTNlNWYzNjU4YjJkNGZmNGM4M2JmMTMzZTgxYmYxNTYwYWIwMmRjYTA2Yzc0MzAxYzcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.QYyzl3bywwyeUlpPHjblahtCPUvNTgXRnqRfvb8OE4w) in the tab title
- New `useTitle` convenience hook created
- Default title of the app (contained in the `<title>` tag in `index.html`) changed from `Web` to `Global Bible Tools`
- All pages in the app were given titles, in the form `[Page] | Global Bible Tools`

## Connected Issues

closes #327 

## QA Steps

1. Go to each page on the app
2. Make sure each tab displays the provided favicon and a reasonable title.

## Post-Deployment
